foundation/foundation-mda/src/main/resources/templates/notifications/notification.sagemaker.docker.pom.vm

```diff
@@ -2,10 +2,8 @@
 Please refer to the documentation at:
 https://github.com/boozallen/aissemble/blob/dev/docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc#sagemaker-training
 
-#if($registryUrlExists)

```

Comment:
I: We are only generating this message if this variable is true, so it was redundant.